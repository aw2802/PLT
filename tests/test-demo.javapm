public class Demo {
	public main void(){

		PrintMe p = new PrintMe(5);
		Tuple<char[], int> returnedTuple = testTuple(3);
		println(returnedTuple<<0>>);
		println(returnedTuple<<1>>);

	}

	public Tuple<char[], int> testTuple(int t){
		Tuple<char[], int> myTuple = new Tuple<char[], int>("Zeynep", t);
		return myTuple;
	}

}

public class PrintMe {
	PrintMe(int x){
		println(x);
	}	
}