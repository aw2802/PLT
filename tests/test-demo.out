5
Zeynep
3